ml-agents/mlagents/trainers/trainer.py

```diff
@@ -70,7 +70,11 @@ def write_tensorboard_text(self, key: str, input_dict: Dict[str, Any]) -> None:
         :param input_dict: A dictionary that will be displayed in a table on Tensorboard.
         """
         try:
-            with tf.Session() as sess:
+            # Prevent GPU memory from being eaten up by this small writing session
+            config = tf.ConfigProto()
```

Comment:
Should we move the ConfigProto setup to a utility function? I'm ok with not doing it now, but if we need it in a 3rd place, I think we should.

Reply:
We can put it in tf_utils, that seems like the natural place for it

Reply:
I moved it to tf in tf_utils - lmk how it looks.